helm/alfresco-content-services/values.yaml

```diff
@@ -160,6 +162,12 @@ ooiService:
     initialDelaySeconds: 10
     periodSeconds: 20
     timeoutSeconds: 10
+  repository:
+    # -- ACS repository host
+    host: acs-alfresco-cs-repository
```

Comment:
That will break in case someone wants to use a release name that's not `acs`. You'll probably need to use a `nameOverride`